services/src/floodlight-modules/src/checkstyle/checkstyle-suppress-known-issues.xml

```diff
@@ -135,19 +135,16 @@
     <suppress files="src/main/java/org/openkilda/floodlight/pathverification/web/PathVerificationServiceWebRoutable.java" lines="19" checks="CustomImportOrder"/>
     <suppress files="src/main/java/org/openkilda/floodlight/pathverification/web/PathVerificationServiceWebRoutable.java" lines="31" checks="JavadocMethod"/>
     <suppress files="src/main/java/org/openkilda/floodlight/statistics/StatisticsService.java" lines="143" checks="MultipleVariableDeclarations"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="20" checks="CustomImportOrder"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="21" checks="CustomImportOrder"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="22" checks="CustomImportOrder"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="40" checks="SummaryJavadoc"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="62" checks="NonEmptyAtclauseDescription"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="84" checks="NonEmptyAtclauseDescription"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="158" checks="SummaryJavadoc"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="166" checks="SummaryJavadoc"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="202" checks="SummaryJavadoc"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="211" checks="SummaryJavadoc"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="220" checks="SummaryJavadoc"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="234" checks="NonEmptyAtclauseDescription"/>
-    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="242" checks="NonEmptyAtclauseDescription"/>
+    <suppress files="src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java" lines="43" checks="SummaryJavadoc"/>
```

Comment:
Fix style violations.

Reply:
This does not belong to this pull request code. I only updated the line number as an extra import comes.

Reply:
This is not in the scope of this pull request and requires extra efforts.